src/libdredd/src/mutate_visitor.cc

```diff
@@ -214,6 +233,20 @@ bool MutateVisitor::TraverseStmt(clang::Stmt* stmt) {
     }
   }
 
+  // Some functions, e.g., __builtin_frame_address(), require their arguments to
+  // be constant integers. Attempting to apply a mutator function to such an
+  // argument would result in a compile-time error. Thus, we record these
+  // arguments' expressions so that they can be rewritten with the values to
+  // which the expressions would normally evaluate.
+  if (const auto* call_expr = llvm::dyn_cast<clang::CallExpr>(stmt)) {
+    const unsigned int frame_address_builtin_id = 455;
+    if (call_expr->getBuiltinCallee() == frame_address_builtin_id) {
```

Comment:
... then you can use use the constant here and get rid of the `frame_address_builtin_id` temporary.

Reply:
done